trade-finance-manager-api/src/graphql/resolvers/query-facilities.js

```diff
@@ -8,17 +10,42 @@ exports.queryAllFacilities = async (queryParams) => {
   const facilities = [];
   // eslint-disable-next-line no-restricted-syntax
```

Comment:
Please do not use inline `linting` rules, they should always be dictated by core `.eslintrc.json` and / or `.prettier.json`. Please remove this line.

Reply:
was there previously